terraform/terraform

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env bash
-set -euo pipefail
-
-pushd "$(dirname "$0")" >/dev/null
-
-# terraform cloud without the remote execution part
-TF_FORCE_LOCAL_BACKEND="1" sops exec-env secrets.yaml "terraform ${*@Q}"
```

Comment:
I don't think we still need `TF_FORCE_LOCAL_BACKEND` but not really sure, there isn't much information about it

Reply:
This isn't needed anymore after 
https://github.com/nix-community/infra/blob/9d68769b4326999bd114b4c08fe0a39b89a49709/terraform/terraform_cloud.tf#L46